operator/gitops/argocd/pipeline-service/tekton-results/base/watcher-logging.yaml

```diff
@@ -0,0 +1,14 @@
+# Adjust logging level of the watcher
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-results-config-logging
+  namespace: tekton-pipelines
```

Comment:
you are correct @enarha thanks for double checking

Reply:
Sure, will do that.

Reply:
I just realized that the namespace name is correct. Because we are patching, the patching happens before the resource is added. And by default, the resource exists in the tekton-pipelines namespace, if we change that, patching will fail.

Reply:
There are different patching methods and they work at different stages. This one in particular happens before.